contrib/devtools/security-check.py

```diff
@@ -12,33 +12,37 @@
 import sys
 import os
 
+from typing import List, Optional
+
 READELF_CMD = os.getenv('READELF', '/usr/bin/readelf')
 OBJDUMP_CMD = os.getenv('OBJDUMP', '/usr/bin/objdump')
 OTOOL_CMD = os.getenv('OTOOL', '/usr/bin/otool')
-NONFATAL = {} # checks which are non-fatal for now but only generate a warning
 
-def check_ELF_PIE(executable):
-    '''
-    Check for position independent executable (PIE), allowing for address space randomization.
-    '''
-    p = subprocess.Popen([READELF_CMD, '-h', '-W', executable], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE, universal_newlines=True)
+def run_command(command) -> str:
+    p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE, universal_newlines=True)
     (stdout, stderr) = p.communicate()
     if p.returncode:
+        print(stderr)
```

Comment:
- Do we want to print `stderr` to `stderr` maybe?
- I'm not sure `Error opening file` is a good error message here. Might want to print the command that failed with the exit status.
- Launching a command and raising an error on non-zero exit status what `subprocess.check_call` does. This might be good enough instead of defining our own function.

Reply:
I've swapped to just using [`subprocess.run`](https://docs.python.org/3.5/library/subprocess.html#subprocess.run).